source/index-management.rst

```diff
@@ -669,6 +669,13 @@ Common API Components
      * filter expression are included in the index.
      */
     partialFilterExpression: Document;
+
+    /**
+     * Optionally specifies a collation to use for the index. If not specified, the default collation
```

Comment:
If you create a collection (using the 'create' command) specifying a collation, will mongod use that collation when creating indexes for that collection?